{
 "errors": {},
 "symbols": [
  "pygeoogc.ArcGISRESTful",
  "pygeoogc.InvalidInputType",
  "pygeoogc.InvalidInputValue",
  "pygeoogc.MatchCRS",
  "pygeoogc.MissingInputs",
  "pygeoogc.RetrySession",
  "pygeoogc.ServerError",
  "pygeoogc.ServiceURL",
  "pygeoogc.ThreadingException",
  "pygeoogc.WFS",
  "pygeoogc.WMS",
  "pygeoogc.ZeroMatched",
  "pygeoogc.__version__",
  "pygeoogc.exceptions",
  "pygeoogc.exceptions.InvalidInputType",
  "pygeoogc.exceptions.InvalidInputValue",
  "pygeoogc.exceptions.MissingInputs",
  "pygeoogc.exceptions.ServerError",
  "pygeoogc.exceptions.ThreadingException",
  "pygeoogc.exceptions.ZeroMatched",
  "pygeoogc.pygeoogc",
  "pygeoogc.pygeoogc.ArcGISRESTful",
  "pygeoogc.pygeoogc.DEF_CRS",
  "pygeoogc.pygeoogc.HTTPURLs",
  "pygeoogc.pygeoogc.RESTfulURLs",
  "pygeoogc.pygeoogc.RetrySession",
  "pygeoogc.pygeoogc.ServiceURL",
  "pygeoogc.pygeoogc.WFS",
  "pygeoogc.pygeoogc.WMS",
  "pygeoogc.utils",
  "pygeoogc.utils.BOX_ORD",
  "pygeoogc.utils.DEF_CRS",
  "pygeoogc.utils.ESRIGeomQuery",
  "pygeoogc.utils.MatchCRS",
  "pygeoogc.utils.bbox_resolution",
  "pygeoogc.utils.check_bbox",
  "pygeoogc.utils.check_response",
  "pygeoogc.utils.threading",
  "pygeoogc.utils.traverse_json",
  "pygeoogc.utils.vsplit_bbox"
 ]
}